src/util-exception-policy-types.h

```diff
@@ -40,6 +40,15 @@ enum ExceptionPolicy {
  * "tcp.reassembly_exception_policy.drop_packet" + 1 */
 #define EXCEPTION_POLICY_COUNTER_MAX_LEN 45
 
+/** exception policy flags */
+#define EXCEPTION_DEFRAG_MEMCAP     BIT_U16(1)
```

Comment:
add a `EXCEPTION_NONE BIT_U16(0)` here?

Reply:
actually just 0, the other bit should start at offset 0
`BIT_U16(0)` is just `1 << 0` == `1`

Reply:
🙇🏽‍♀️